src/host/getset.cpp

```diff
@@ -1291,6 +1291,25 @@ void ApiRoutines::GetConsoleDisplayModeImpl(ULONG& flags) noexcept
     return STATUS_SUCCESS;
 }
 
+// Routine Description:
+// - A private API call for setting the ENABLE_WRAP_AT_EOL_OUTPUT mode.
+//     This controls whether the cursor moves to the beginning of the next row
+//     when it reaches the end of the current row.
+// Parameters:
+// - wrapAtEOL - set to true to wrap, false to overwrite the last character.
+// Return value:
+// - STATUS_SUCCESS if handled successfully.
+[[nodiscard]] NTSTATUS DoSrvPrivateSetAutoWrapMode(const bool wrapAtEOL)
+{
+    auto& gci = ServiceLocator::LocateGlobals().getConsoleInformation();
+    auto& outputMode = gci.GetActiveOutputBuffer().GetActiveBuffer().OutputMode;
+    if (wrapAtEOL)

```

Comment:
Please use braces, even for one line if/else.

Reply:
Alternatively, you can use `WI_UpdateFlag` here.

Reply:
Yeah, that's much better. Thanks.